src/modules/layout/components/Header.js

```diff
@@ -123,6 +124,24 @@ class Header extends React.PureComponent<PropsType, StateType> {
     this.getNavigation().navigate('ChangeLanguageModal')
   }
 
+  onShare = async () => {
+    const url = 'https://integreat-app.de/'
+    const message: string = Platform.select({
```

Comment:
Is there a reason why we can't display the same message on both platforms?

Reply:
I read this thread https://expo.canny.io/feature-requests/p/share-api-support-for-sharing-urls-on-android
but I didn't have a chance to test it on iOS

Reply:
Alright I think this is the way to go on ios vs android.

Only i18n is missing for the texts. You can add a constant to this file: https://docs.google.com/spreadsheets/d/1kMxVlqzwjjDoARO5Ss5oMNUEjqubT0Ps8NB7cguP-yU/

Then just add a key to the translation to locales.json
Also create a task to add the translations to the project once the Konstanten sheet has been translated.

Reply:
This is still relevant.